src/local_pathfinding/local_pathfinding/ompl_path.py

```diff
@@ -225,7 +225,7 @@ def _init_simple_setup(self, local_path_state, land_multi_polygon) -> og.SimpleS
         start_x = start_position_in_xy.x
         start_y = start_position_in_xy.y
 
-        goal_position = self.state.global_path.waypoints[-1]
+        goal_position = self.state.reference_latlon
         goal_position_in_xy = cs.latlon_to_xy(self.state.reference_latlon, goal_position)
         goal_polygon = self.create_buffer_around_position(goal_position_in_xy)
         goal_x, goal_y = goal_position_in_xy
```

Comment:
Since the reference latlon and global waypoint (goal position) are all the same thing, `goal_x` and `goal_y` will both always be 0. Ok if I take this out?